.phan/config.base.php

```diff
@@ -146,15 +148,18 @@ function make_phan_config( $dir, $options = array() ) {
 					// Most of these are probably from our intra-monorepo symlinks.
 					'(?:jetpack_)?vendor/.*/(?:wordpress|(?:jetpack_)?vendor|node_modules)/',
 					// Yoast/phpunit-polyfills triggers a lot of PhanRedefinedXXX errors.
-					// Avoid that by excluding certain files.
-					'vendor/yoast/phpunit-polyfills/src/Polyfills/.*_Empty\.php',
-					'vendor/yoast/phpunit-polyfills/src/TestCases/TestCasePHPUnitGte8\.php',
+					// Avoid that by excluding the versions of the files for PHPUnit < 9.6
+					'vendor/yoast/phpunit-polyfills/src/Polyfills/(?!.*_Empty).*\.php',
+					'vendor/yoast/phpunit-polyfills/src/TestCases/TestCasePHPUnitLte7\.php',
```

Comment:
Should we bother running `phan` in any vendor files, given we probably wouldn't be modifying them anyway? 🤔

Reply:
Phan effectively has three ways it can handle a file:

1. Ignore it entirely.
2. Parse and analyze it.
3. Parse it, so it knows what functions and such it defines, but don't analyze it.

Vendor files need to be parsed so Phan knows what functions and classes they provide, but not analyzed since we can't do anything to fix them.

Phan's configuration work out like this, BTW:

1. `directory_list` and `file_list` specify the files to parse and (by default) analyze.
2. `exclude_file_list` and `exclude_file_regex` allow for exceptions to the usually-blanket coverage from `directory_list`.
3. `exclude_analysis_directory_list` lists dirs/files to skip analyzing.

Reply:
Thanks, that helps! I see `vendor` is in `exclude_analysis_directory_list`.